src/Microsoft.AspNet.Authentication.Cookies/Events/CookieRedirectContext.cs

```diff
@@ -29,5 +30,7 @@ public CookieRedirectContext(HttpContext context, CookieAuthenticationOptions op
         /// </summary>
         [SuppressMessage("Microsoft.Design", "CA1056:UriPropertiesShouldNotBeStrings", Justification = "Represents header value")]
         public string RedirectUri { get; set; }
+
+        public AuthenticationProperties Properties { get; set; }
```

Comment:
My plan was to wait and do a full pass, lots of doc comments are missing/out of date currently in this repo...  This issue is tracking all up https://github.com/aspnet/Security/issues/190